lms/templates/seq_module.html

```diff
@@ -17,7 +17,6 @@
            data-id="${item['id']}"
            data-element="${idx+1}"
            href="javascript:void(0);"
-           title="${item['title']|h}"
            data-page-title="${item['page_title']|h}"
```

Comment:
@dianakhuang do you recall why this was added in? (git blame shows you, during the a/b testing work Spring 2014)